auxiliary/OpGasPriceOracle/contracts/OpGasPriceOracle.sol

```diff
@@ -74,16 +74,19 @@ contract OpGasPriceOracle is IExternalNode {
     ) internal view returns (uint256 costOfExecutionGrossEth) {
         IOVM_GasPriceOracle ovmGasPriceOracle = IOVM_GasPriceOracle(ovmGasPriceOracleAddress);
 
-        uint256 gasPriceL2 = ovmGasPriceOracle.gasPrice();
-        uint256 overhead = ovmGasPriceOracle.overhead();
-        uint256 l1BaseFee = ovmGasPriceOracle.l1BaseFee();
+        uint256 gasPriceL2 = block.basefee;
```

Comment:
i understand that it leads to the same, but for the sake of consistency, maybe we need change the pull here to `ovmGasPriceOracle.baseFee` in case optimism decides to change the logic of  this parameter?

Reply:
i will provide the updated gasUnitsL1 and gasUnitsL2 parameters a bit later this week for settlement and liquidations.